test/cql-pytest/test_logs.py

```diff
@@ -113,3 +116,31 @@ def test_log_table_operations(cql, test_keyspace, logfile):
     with new_test_table(cql, test_keyspace, 'p int PRIMARY KEY') as table:
         wait_for_log(logfile, f'Creating {table}')
     wait_for_log(logfile, f'Dropping {table}')
+
+
+@pytest.fixture(scope="module")
+def table_batch(cql, test_keyspace):
+    table = test_keyspace + "." + unique_name()
+    cql.execute(f"CREATE TABLE {table} (k int primary key, t text)")
+    yield table
+    cql.execute("DROP TABLE " + table)
+
+
+def test_warning_message_for_batch_exceeding_warning_threshold(cql, table_batch, logfile):
+    """Batch size above threshold should add warning message to scylla logs"""
+    warning_pattern = r"WARN .+BatchStatement - Batch modifying [\d]+ partitions in [_\w.]+ is of size [\d]+ bytes, "\
+                      r"exceeding specified WARN threshold of"
+    cql.execute(generate_big_batch(table_batch, 256))
+    wait_for_log(logfile, warning_pattern, timeout=1)
+
+
+def test_error_message_for_batch_exceeding_fail_threshold(cql, table_batch, logfile):
+    """Batch size above threshold should add error message to scylla logs"""
+    try:
+        cql.execute(generate_big_batch(table_batch, 1025))
+    except InvalidRequest:
```

Comment:
Why not using pytest.rasie here ? we expect it to fail.
After that we can wait for the logs.

Reply:
no very specific/urgent reason - with try-catch I could add quite clear reason of failure instead of default one that pytest raises throws.

Reply:
I don't think we need this test. The test that this request failed you already added in test_batch.py. What this test "adds" is the fact there is no warning in the log, but 1. it's not surprising that we don't print a warning in case an error, and 2. there is no real reason why NOT to print such a warning - we could have printed a log message in this case too... I'm not sure we want to enshrine the fact that we don't.

Reply:
This test goal is to test log messages - `ERROR` message should appear for batch too big. `test_batch.py` does not look into logs, so maybe it looks like duplicate, but it isn't, as intention is different.